scripts/build_utils.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-VENV="$WORKSPACE/venv/bin"
+VENV="/tmp/jenkins-venv/bin"
```

Comment:
You are right, I should have been more considerate of Shadowman's feelings. Will fix.